index.html

```diff
@@ -169,7 +169,7 @@ <h2 id="main-h2">Private Digital Currency</h2>
                         <div class="row center-xs">
                             <div class="col">
                                 <h2>User Guides</h2>
-                                <p class="main-resources">Step-by-step guides to all things Monero are separated by category and cover everything from creating a wallet to mnemonic seeds, and even how to edit this website.</p>
+                                <p class="main-resources">Step-by-step guides to all things Monero are separated by category and cover everything from creating a wallet to recovering a wallet, and even how to edit this website.</p>
```

Comment:
`creating a wallet to recovering a wallet`

Maybe `create a wallet to setting up a node`, so it shows a dichotomy between easy informational stuff and more technical stuff.

Reply:
Yes!  Thats what I wanted to do! :D :D  great change.

Reply:
I think 'node' may even be too abstract for the main page, so I used `creating a wallet to supporting the network'.  This hits manys goals; it creates the dichotomy, informs the reader they as an individual can help Monero and likewise Monero is supported by users like them.  

Also it ties into the text from the 'Monero is Secure' section: `... cash powered by a network operated by its users`.